frontend/app/views/info/elevatedSupporter.scala.html

```diff
@@ -9,8 +11,83 @@
 @main("Supporters", pageInfo=pageInfo, countryGroup=Some(countryGroup)) {
 
     <section class="elevated-supporter">
-        @fragments.page.elevatedBecomeSupporter(heroImage, supporterPlans, countryGroup, showCurrencyPrefix = false)
-        @fragments.page.elevatedCloserTheGuardian(supporterPlans, countryGroup, showCurrencyPrefix = false)
-        @fragments.page.elevatedWhySupport(detailImage, supporterPlans, countryGroup, showCurrencyPrefix = false)
+
+        @* ===== First part ===== *@
+        @fragments.page.elevatedBecomeSupporter(heroImage, supporterPlans, countryGroup, showCurrencyPrefix = false, Html("Become a<br>Guardian Supporter"), Html("Be part of the Guardian's future, by helping to secure it")
+        ){
+            <div class="elevated-become-supporter-body__col u-content-width--left">
+                <p class="u-responsive-p">
+                    We want to make the world a better, fairer place. We want to keep the powerful honest. And we believe that doing so means keeping society informed by producing quality, independent journalism, which discovers and tells readers the truth.
+                </p>
+                <p class="u-responsive-p">
+                    It’s essential for the functioning of democracy. And our unique ownership structure means no one can tell us to censor or drop a story.
+                </p>
+            </div>
+
+            @fragments.page.elevatedBecomeSupporterBodyVideo(Videos.scottTrustExplained, "Katharine Viner, editor-in-chief, explains the Guardian's unique ownership model")
+
+            <div class="elevated-become-supporter-body__col u-content-width--left">
+                <p class="u-responsive-p">
+                    But it’s difficult and expensive work. While more people are reading the Guardian than ever before, far fewer are paying for it. And advertising revenues across the media are falling fast.
+                </p>
+                <p class="u-responsive-p">
+                    So if you read us, if you like us, if you value our perspective – then become a Supporter and help make our future more secure.
+                </p>
+            </div>
+        }
+
+        @* ===== Second Part ===== *@
+        @fragments.page.elevatedCloserTheGuardian(supporterPlans, countryGroup, showCurrencyPrefix = false, Html("Supporters get closer<br>to the Guardian")){
+            <div class="elevated-closer-guardian-body__col elevated-closer-guardian-body__text u-content-width--right">
+                <div class="elevated-closer-guardian-body__list">
+
+                    <p class="u-responsive-p">
+                        As a Guardian Supporter, you’ll enjoy a number of benefits, including:
+                    <p>
+
+                    <ul>
+                        <li class="elevated-closer-guardian-body__list_item">
+                            Exclusive emails from Guardian journalists
```

Comment:
Thank you for spotting that @rtyley. I created a card (https://trello.com/c/njNyDm0A/222-refactor-benefits-for-landing-page) to refactor this in this sprint.